README.md

```diff
@@ -48,32 +41,123 @@ So that I am reassured that my order will be delivered on time
+  end
+end
+```
+I then proceeded to write tests to actually create and output the menu and the supporting code for that. I removed the respond to method expectation as it seemed redundant; I knew it would expect that and need a method to be able to work.
+
+**Tests:**  
+```ruby
+describe Menu do
+  describe '#initialize' do
+    it 'initializes with the list of dishes' do
+      expect(menu.dishes).to eq @dishes
+    end
+  end
+
+  describe '#read_menu' do
+    it 'outputs the menu' do
+      expect { menu.read_menu }.to output(@dishes).to_stdout
+    end
+  end
+end
+```
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+**Code:**  
+```ruby
+class Menu
+  attr_reader :dishes
+
+  def initialize
+    create_menu
+  end
+
+  def create_menu
+    @@dishes = { 'Kobe Slider' => 7,
+                 'Shrimp Tempura' => 4,
+                 'Salmon Teriyaki' => 5,
+                 'Chicken Katsu' => 8,
+                 'Edamame Pot' => 2 }
+  end
+
+  def read_menu
+    @@dishes.each do |key, value|
+      puts key + ": £" + value.to_s
+    end
+  end
+end
+```
 
-Notes on Test Coverage
+Creating the order
 ------------------
+In much the same way as the menu class, the order class was tested and created following the TDD method.
+
+
+  **Ordering the dish:**  
+  When a customer orders a dish, I set the default balance to equal 1 under the assumption that the customer would order at least 1 of any of the dishes. This quantity is also changeable at the time of ordering. The dish is looked up (from the menu list hash from the menu class) to retrieve the name and price key-value pair and adds it to the basket. Each combination of dishes ordered is grouped and pushed to an updated basket whilst also being displayed to the customer as being added to the basket.
+
+  **Showing the basket:**  
+  The basket shows the customer all the dishes and prices they have ordered and gives them a total price to pay.
+
+  **Placing the order:**  
+  Once the customer is satisfied with their choices, they can place the order which sends them a text that thanks them for their order, with their name and what time it will be delivered (I set it to deliver an hour after the order was placed).
+
+  [I used a Twilio number and my phone number to test the text(s) deliveries could be received.]
+
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you run your tests.
+How I plan to finish the challenge
+----------------------------------
+Although, I have met the requirements from the user stories, I would like to incorporate some doubles, mocks and stubs into my program. I found it difficult to add these without the tests breaking, even though the code was working correctly.
```

Comment:
Definitely worth lasering into why you found this tricky. Was it because there were too many responsibilities in one class? Was it understanding the errors? Was it debugging your way through several errors to get the code working? Was it understanding how to use doubles? Or was it something else?